uit/uit.py

```diff
@@ -15,10 +15,10 @@
 
 import dodcerts
 import requests
-import yaml
 from flask import Flask, request, render_template_string
 from werkzeug.serving import make_server
 
+from .config import parse_config, DEFAULT_CA_FILE
```

Comment:
Adding DEFAULT_CONFIG for the next comment on line 96:
```suggestion
from .config import parse_config, DEFAULT_CA_FILE, DEFAULT_CONFIG
```